ibis/expr/types/geospatial.py

```diff
@@ -687,6 +688,11 @@ def convert(
         -------
         GeoSpatialValue
             Transformed geometry
+
+        See Also
+        --------
+        [`flip_coordinates`](#ibis.expr.types.geospatial.GeoSpatialValue.flip_coordinates)
```

Comment:
Is this the right way to reference a "see also"? I followed another example, I wasn't quite sure it works, have docs build broken locally at the moment, but didn't want to delayed pushing this.